tezos-sig-gating-contracts/contracts/examples/gatedNftClaimer.mligo

```diff
@@ -1,7 +1,6 @@
-// #import "../../.ligo/source/i/ligo__s__fa__1.4.2__ffffffff/lib/main.mligo" "FA2"
-// #import "../../.ligo/source/i/nexeraid__s__sig_gating__1.0.2__ffffffff/lib/main.mligo" "SigGatedExtendable"
 #import "@ligo/fa/lib/main.mligo" "FA2"
 #import "@nexeraid/sig-gating/lib/main.mligo" "SigGatedExtendable"
+// #import "../../tezos-lib-sig-gating-extendable/lib/main.mligo" "SigGatedExtendable"
```

Comment:
I left this comment because I often use it during dev. if the Ligolang registry is down , I need to point to the source coe of the library in order to be able to compile contracts.